lib/http/server.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
```

Comment:
Bikeshed: Just calling this `http/server` is likely to be fairly confusing. I'd go with something like `http/static-server`.

Reply:
The longer the name, the less practical it is for using in CLI, the name was inspired of `python -m http.server`. I've thought about using `node:http/1.1`, as we are discussing having a built-in h3/quic version of that. Regarding your `http/static-server` suggestion, for consistency with e.g. `node:child_process`  and `node:worker_threads`, should we use an underscore rather than a dash?

Reply:
If we want something shorter, `http/static` would work, I think.